src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int bound = 100;

```

Comment:
The `bound` variable is defined as a static field, which is good for a constant. However, it should be declared as `final` to indicate that it is a constant value and should not be changed.